runtime/near-evm-runner/tests/standard_ops.rs

```diff
@@ -0,0 +1,241 @@
+    assert_eq!(context.get_balance(sub_addr).unwrap(), U256::from(100));
+}
+
+#[test]
+fn test_contract_to_eoa_transfer() {
+    let (mut fake_external, test_addr, vm_config, fees_config) = setup_and_deploy_test();
+
+    let (input, _) = soltest::functions::return_some_funds::call();
+    let mut context =
+        create_context(&mut fake_external, &vm_config, &fees_config, accounts(1), 100);
+    let raw = context.call_function(encode_call_function_args(test_addr, input)).unwrap();
+
+    let sender_addr = raw[12..32].to_vec();
+    assert_eq!(context.get_balance(test_addr.0.to_vec()).unwrap(), U256::from(150));
+    assert_eq!(context.get_balance(sender_addr).unwrap(), U256::from(50));
+}
+
+#[test]
+fn test_get_code() {
+    let (mut fake_external, test_addr, vm_config, fees_config) = setup_and_deploy_test();
+    let context = create_context(&mut fake_external, &vm_config, &fees_config, accounts(1), 0);
+
+    assert!(context.get_code(test_addr.0.to_vec()).unwrap().len() > 1500); // contract code should roughly be over length 1500
+    assert_eq!(context.get_code(vec![0u8; 20]).unwrap().len(), 0);
+}
+
+#[test]
+fn test_view_call() {
+    let (mut fake_external, test_addr, vm_config, fees_config) = setup_and_deploy_test();
+
+    // This should NOT increment the nonce of the deploying contract
+    // And NO CODE should be deployed
+    let (input, _) = soltest::functions::deploy_new_guy::call(8);
+    let mut context = create_context(&mut fake_external, &vm_config, &fees_config, accounts(1), 0);
+    let raw = context
+        .view_call_function(encode_view_call_function_args(
+            test_addr,
+            test_addr,
+            U256::from(0),
+            input,
+        ))
+        .unwrap();
+    assert_eq!(context.get_nonce(test_addr.0.to_vec()).unwrap(), U256::from(0));
+
+    let sub_addr = raw[12..32].to_vec();
+    assert_eq!(context.get_code(sub_addr).unwrap().len(), 0);
+}
+
+#[test]
+fn test_solidity_accurate_storage_on_selfdestruct() {
+    let (mut fake_external, vm_config, fees_config) = setup();
+    let mut context =
+        create_context(&mut fake_external, &vm_config, &fees_config, accounts(1), 100);
+    assert_eq!(
+        context.deposit(near_account_id_to_evm_address(&accounts(1)).0.to_vec()).unwrap(),
+        U256::from(100)
+    );
+
+    // Deploy CREATE2 Factory
+    let mut context = create_context(&mut fake_external, &vm_config, &fees_config, accounts(1), 0);
+    let factory_addr = context.deploy_code(hex::decode(&FACTORY_TEST).unwrap()).unwrap();
+
+    // Deploy + SelfDestruct in one transaction
+    let salt = H256([0u8; 32]);
+    let destruct_code = hex::decode(&DESTRUCT_TEST).unwrap();
+    let input = create2factory::functions::test_double_deploy::call(salt, destruct_code.clone()).0;
+    let raw = context.call_function(encode_call_function_args(factory_addr, input)).unwrap();
+    assert!(create2factory::functions::test_double_deploy::decode_output(&raw).unwrap());
+}
```

Comment:
Add test incorrect nonce?

Reply:
There is no way to provide nonce to evm.